README.md

```diff
@@ -1,5 +1,66 @@
+
+This is the repository for the **official app of Shishir 2020**
+
+[**Download**](https://play.google.com/store/apps/details?id=nitmeghalaya.shishir2020&hl=en_IN "Cognitia 2018 | App on Play Store")
+
+<!--
+[![Alt](https://img.shields.io/badge/downloads-1-green.svg?style=for-the-badge)](https://play.google.com/store/apps/details?id=nitmeghalaya.shishir2020&hl=en_IN "Shishir 2020 | App on Play Store")
+[![Alt](https://img.shields.io/badge/rating-NULL%2F5%20(num_ratings)-ff69b4.svg?style=for-the-badge)](https://play.google.com/store/apps/details?id=nitmeghalaya.shishir2020&hl=en_IN "Shishir 2020 | App on Play Store")
+-->
+[![Alt](https://img.shields.io/badge/Android-5.0%2B-red.svg?style=for-the-badge)](https://play.google.com/store/apps/details?id=nitmeghalaya.shishir2020&hl=en_IN "Shishir 2020 | App on Play Store")
+[![Alt](https://img.shields.io/badge/Shishir-2020-brightgreen.svg?style=for-the-badge)](http://cognitia.nitmeghalaya.in/ "Shishir 2020")
+[![Alt](https://img.shields.io/badge/NIT-Meghalaya-blue.svg?style=for-the-badge)](http://2018nitm.nitmeghalaya.in/ "NIT Meghalaya")
+[![Alt](https://img.shields.io/badge/using-AndroidX-brightgreen.svg?style=for-the-badge)](#)
+[![Alt](https://img.shields.io/badge/Built%20with-Kotlin-purple.svg?style=for-the-badge)](#)
+[![Alt](https://img.shields.io/badge/App%20size-7.1%20MB-orange.svg?style=for-the-badge)](https://play.google.com/store/apps/details?id=nitmeghalaya.shishir2020&hl=en_IN "Shishir 2020 | App on Play Store")
+
+
+## Features
+
+* Information about various events happening during Shishir
+* Details of members of various teams.
+* Sechedule of the events.
+* Registration forms of different events
+* Details about Sponsors
+* A navigation drawer that facilitates navigating through the app.
+* Links to other social media accounts of Shishir
+
+
+## Screenshots
+
+<img src="https://lh3.googleusercontent.com/5h8pGkXBXbqEkvSfTVvSDT3nEvNWMzWcrH9Jp7i-FPwOum0XepBojKGfjdyhpf5H8yMN=w1536-h759-rw" alt="Screenshot1" width="320"/>
+<img src="https://lh3.googleusercontent.com/KiRgfZoPqX41YJZsexeNWcq6BS80cvDtx7LJ387bHL9dJq7AfTveJdoHFJTCROF9QA=w1536-h759-rw" alt="Screenshot2" width="320"/>
+
+
+## Development
+ The app makes use of the following technologies:
+ 
+ * Kotlin over traditional Java in Android Development
+ * App makes use of the new and improved `AndroidX` libraries in place of the `Support Libraries`
+ * Firebase Realtime Database
+ * Firebase UI (Uses Firebase Recycler Adapter)
+ * Firebase Cloud Messaging (FCM)
+ * Lottie (for displaying animations in intro slider)
+ * Shape of View (to modify view shapes)
+ * Toast Splitties (Kotlin Extension)
+ * Timber (extensible to provide utility over Android's Log class)
+
+And a couple more!
```

Comment:
You can add more info. like architecture patterns and other Jetpack components. Please refer [this repo](https://github.com/Devansh-Maurya/PukaPuka#technologies-used) or [this one](https://github.com/Devansh-Maurya/TreeCare#technologies-used)

Reply:
Okay I'll fix this along with the other one

Reply:
Thanks